### ISSUE TITLE
Multiple type fields don`t work at Android OS in my phone

### ISSUE THREAD

**Issue by DanielTeacher1** [r0uv3n/Multiple-type-fields-on-Card-for-Anki-2.1#15]
![Screenshot_20241230-173715_AnkiDroid](https://github.com/user-attachments/assets/8c870eff-13da-4737-86da-bf4eebd21b98)
![Screenshot_20241230-173756_AnkiDroid](https://github.com/user-attachments/assets/63940597-faa1-463f-88f7-fa3d5476a473)
However, it works Okay in my computer.
What to do?

**Comment by MaximKa99** [comment]
anki addons do not work on Android

**Comment by unit-of-inductance** [comment]
Yeah, sorry for only getting back to you now, but since you cannot install this Addon on your phone, you also won't be able to use its functionalities there.
